Cargo.toml

```diff
@@ -13,7 +13,10 @@ homepage = "https://github.com/ackotech/realm"
 
 [features]
 default = []
-dynamic-elm-dependencies = []
+postgres = ["diesel/postgres"]
```

Comment:
Add in readme that enabling either of postgres+postgres_default or sqlite+sqlite_default features is now required.